build.sbt

```diff
@@ -8,9 +8,7 @@ organization := "com.codacy"
 scalaVersion := scala212
 crossScalaVersions := Seq(scala211, scala212, scala213)
 
-libraryDependencies ++= Seq(
-  "org.specs2" %% "specs2-core" % "4.6.0" % Test
-)
+libraryDependencies ++= Seq("org.specs2" %% "specs2-core" % "4.6.0" % Test)
```

Comment:
If it's easy you can bump specs2 to 4.7.1

Reply:
Done